pandas/io/msgpack/_packer.pyx

```diff
@@ -235,7 +235,8 @@ cdef class Packer:
                 continue
             else:
                 raise TypeError("can't serialize {thing!r}".format(thing=o))
-            return ret
+            break
```

Comment:
really? cython doesn't understand the else here?

Reply:
Its clang giving a "code will never be executed" warning